src/ripple/app/misc/impl/ValidatorList.cpp

```diff
@@ -170,6 +170,12 @@ ValidatorList::load (
             return false;
         }
 
+        if (publicKeyType(*id)!= KeyType::secp256k1)
+        {
+            // Trusted ed25519 keys will be added to the manifest cache
+            continue;
+        }
```

Comment:
This has a smell to me. I can't find any other checks of the key type in `ValidatorList`, which would imply that an ed25519 key is currently a valid Validator key. I know none of our Validators are using them, but it feels like we're downgrading functionality here.

PS. Nit: need a space before `!=`.